src/Route/Route.php

```diff
@@ -29,13 +35,15 @@ class Route implements RouteInterface
 
 
     /**
+     * @param string $name The name of the route.
      * @param string $uri The URI.
      * @param string $controller The controller name.
      * @param string $action The controller action name.
      * @param string $requestType The request type.
      */
-    public function __construct($uri, $controller, $action, $requestType = 'GET')
+    public function __construct($uri, $controller, $action, $requestType = 'GET', $name = null)
```

Comment:
I am thinking if its best to have the name as the first parameter and release a major version for this?